docs/lookups.rst

```diff
@@ -235,6 +235,37 @@ and then assign UserData in a LaunchConfiguration or Instance to self.get_variab
 Note that we use AWSHelperFn as the type because the parameterized-b64 codec returns either a
 Base64 or a GenericHelperFn troposphere object.
 
+.. _ssmstore:
+
+SSM Parameter Store Lookup
+----------
+
+The ``ssmstore`` lookup type retrieves a value from the Simple Systems
+Manager Parameter Store.
+
+As an example, if you have a database and it has a parameter called
+``DBUser`` that you don't want to store in clear text in your config,
+you could instead store it as a SSM parameter named ``MyDBUser``.
+
+For example::
+
+  # We use the aws cli to store the d
+  $ aws ssm put-parameter --name "MyDBUser" --type "String" \
+      --value "root"
+
+  # In stacker we would reference the value like:
+  DBUser: ${ssmstore us-east-1@MyDBUser}
+
+  # The above would resolve to
+  DBUser: root
+
+Encrypted values ("SecureStrings") can also be used, which will be
```

Comment:
It might be worth mentioning here that if you use the `ssmstore` lookup for encrypted secrets, the value will be stored in plaintext across AWS services (CloudFormation, S3, etc), so use it wisely (e.g. ensuring that you use CFNString with `no_echo` when passing it down, etc).